server/src/test/java/org/solarbank/server/integration/EnergySavingControllerIT.java

```diff
@@ -81,14 +87,18 @@ public void energySavingController_emptyRequest_status400() {
     @Test
     public void energySavingController_throwException_status500() {
```

Comment:
This test uses a mock response for the nasaData but think it might still be making an API call...